backend/src/main/java/com/ody/mate/domain/EtaStatus.java

```diff
@@ -0,0 +1,10 @@
+package com.ody.mate.domain;
+
+public enum EtaStatus {
+    LATE_WARNING,
+    ARRIVAL_SOON,
+    ARRIVED,
+    LATE,
+    MISSING,
```

Comment:
[제안]
정렬해주면 어떨까요? 왜냐면.. 기부니 좋으니까요😄
```suggestion
    ARRIVAL_SOON,
    ARRIVED,
    LATE_WARNING,
    LATE,
    MISSING,
```